.github/workflows/lint_and_integration.yaml

```diff
@@ -151,7 +151,7 @@ jobs:
     name: Charm e2e tests - ${{ matrix.arch }}
     strategy:
       matrix:
-        arch: [amd64]
+        arch: [amd64, arm64]
```

Comment:
the e2e tests start LXD containers as VMs right?

i see this failure on arm runners b/c they don't support virtualization.  We run into the same problem with lxd on arm in k8s-operator.  FYI -- this might not be possible

```
Failed creating instance record: Instance type "virtual-machine" is not supported on this server: KVM support is miss...
```